src/app/rulers/rulersActions.js

```diff
@@ -0,0 +1,24 @@
+export const EDIT_RULER = 'EDIT_RULER'
+export const MOVE_CURRENT_RULER = 'MOVE_CURRENT_RULER'
+export const TOGGLE = 'TOGGLE'
```

Comment:
Isn't this action too generic ? also we could have namespacing issues, so I'd prefer having something like `RULER_TOGGLE`

Reply:
same for `RESET`

Reply:
👍